pkg/syncer/volume_health_reconciler.go

```diff
@@ -355,7 +355,7 @@ func (rc *volumeHealthReconciler) syncPVC(key string) error {
 	for _, tkgPV := range tkgPVList {
 		// Update Tanzu Kubernetes Grid PVC volume health annotation
 		if tkgPV.Spec.ClaimRef == nil {
-			log.Debugf("skipping tkgPV: %v with nil ClaimRef", tkgPV.Name)
+			log.Infof("skipping tkgPV: %v with nil ClaimRef", tkgPV.Name)
```

Comment:
We could have unbound PVs in the system. Does it mean it gets logged at info level all the time? Would that fill up the CSI logs?